webpack/components/FormShow.js

```diff
@@ -46,9 +46,18 @@ class FormShow extends Component {
     return (
       <div className="col-md-9 nopadding maincontent">
         <div className="action_bar no-print">
-          {this.props.currentUser && this.props.currentUser.id && form.mostRecent == form.version &&
+          {this.isPublishable(form) &&
+              <a className="btn btn-default" href="#" onClick={() => {
+                this.props.publishForm(form.id);
+                return false;
+              }}>Publish</a>
```

Comment:
@eedrummer I know this was discussed yesterday: is after publish a form redirecting to the dashboard the intended behavior? This is different from questions where publishing just changes the display of the show page (wondering if this is better / we should add a status descriptor with either Draft / Published to the show page)